src/main/java/com/github/messenger4j/send/templates/ListTemplate.java

```diff
@@ -0,0 +1,296 @@
+                this.listTemplateBuilder = listTemplateBuilder;
+                this.elements = new ArrayList<>();
+            }
+
+            public Builder addElement(String title) {
+                return new Builder(title, this);
+            }
+
+            public ListTemplate.Builder done() {
+                Assert.sizeNotGreaterThan(this.elements, MAX_ELEMENTS, "elements", IllegalStateException.class);
+                Assert.sizeNotLessThan(this.elements, MIN_ELEMENTS, "elements", IllegalStateException.class);
+                return this.listTemplateBuilder.elements(Collections.unmodifiableList(new ArrayList<>(this.elements)));
+            }
+
+            public void addElementToList(Element element) {
+                this.elements.add(element);
+            }
+        }
+
+        public static final class Builder {
+            private static final int BUTTONS_LIMIT = 1;
+            private final String title;
+            private final ListBuilder listBuilder;
+            private String subtitle;
+            private String imageUrl;
+            private List<Button> buttons;
+            private DefaultAction defaultAction;
+
+            public Builder(String title, ListBuilder listBuilder) {
+                this.title = title;
+                this.listBuilder = listBuilder;
+            }
+
+            public Builder subtitle(String subtitle) {
+                this.subtitle = subtitle;
+                return this;
+            }
+
+            public Builder imageUrl(String imageUrl) {
+                this.imageUrl = imageUrl;
+                return this;
+            }
+
+            public Builder buttons(List<Button> buttons) {
+                Assert.notNullOrEmpty(buttons, "buttons");
+                Assert.sizeNotGreaterThan(buttons, BUTTONS_LIMIT, "buttons", IllegalStateException.class);
+                this.buttons = buttons;
+                return this;
+            }
+
+            public DefaultAction.Builder defaultAction(String url) {
+                return new DefaultAction.Builder(url, this);
+            }
+
+            private Builder defaulAction(DefaultAction defaultAction) {
+                this.defaultAction = defaultAction;
+                return this;
+            }
+
+            public ListBuilder build() {
+                listBuilder.addElementToList(new Element(this));
+                return listBuilder;
+            }
+
+        }
+
+        public static final class DefaultAction {
+
+            private final Button.ButtonType type;
+            private final String url;
+            @SerializedName("webview_height_ratio")
+            private final WebviewHeightRatio webviewHeightRatio;
+
+            public DefaultAction(Builder builder) {
+                this.url = builder.url;
+                this.webviewHeightRatio = builder.webviewHeightRatio;
+                this.type = Button.ButtonType.URL;
+            }
+
+            public enum WebviewHeightRatio {
```

Comment:
I was considering it but then I decided to use a separete enum. I've changed it now. 
The spec explicitly says that the DefaultAction has the same attributes as UrlButton so I guess it is ok if we reference it in our code.